### PR TITLE
1.2.4 version

### PR DESCRIPTION
**Pull request by remstone7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +30,7 @@ Fixed
 
 
 [Unreleased]: https://github.com/klaviyo/magento2-klaviyo/compare/1.2.3...HEAD
```
Comment (smoucka) [inline CHANGELOG.md:32]:
```suggestion
[Unreleased]: https://github.com/klaviyo/magento2-klaviyo/compare/1.2.4...HEAD
```

Comment (jordanallain) [inline CHANGELOG.md:32]: Should this be 1.2.4...HEAD now?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### [Unreleased]
+
+### [1.2.4] - 2020-12-01
+
 Added
```
Comment (smoucka) [inline CHANGELOG.md:11]: nit: looks like we lost the formatting on these headers
```suggestion
##### Added
```

**Review by smoucka — APPROVED** [review]
Two little formatting things

File: CHANGELOG.md (hunk 1 above)
Comment (remstone7) [reply CHANGELOG.md:32]: yup, missed this

**Pull request merged**